examples/angular/src/pages/ui/primitive-poc/view/view.component.html

```diff
@@ -0,0 +1,99 @@
+<button
+  amplify-view
+  ariaLabel="View example"
+  isDisabled="true"
+  backgroundColor="var(--amplify-colors-white)"
+  borderRadius="6px"
+  border="1px solid var(--amplify-colors-black)"
+  boxShadow="3px 3px 5px 6px var(--amplify-colors-neutral-60)"
+  color="var(--amplify-colors-blue-60)"
+  height="4rem"
+  maxWidth="100%"
+  padding="1rem"
+  width="20rem"
+  fullWidth="true"
+  (click)="clickbutton('Button')"
+>
+  'I'm a button! 🤩
+</button>
+<br />
+
+<p
+  amplify-view
+  ariaLabel="View example"
+  isDisabled="true"
+  backgroundColor="var(--amplify-colors-white)"
+  borderRadius="6px"
+  border="1px solid var(--amplify-colors-black)"
+  boxShadow="3px 3px 5px 6px var(--amplify-colors-neutral-60)"
+  color="var(--amplify-colors-blue-60)"
+  height="4rem"
+  maxWidth="100%"
+  padding="1rem"
+  width="20rem"
+  fullWidth="true"
+  (click)="clickbutton('p')"
+>
+  'I'm a p! 🤩
+</p>
+<br />
+
+<span
+  amplify-view
+  ariaLabel="View example"
+  backgroundColor="var(--amplify-colors-white)"
+  borderRadius="6px"
+  border="1px solid var(--amplify-colors-black)"
+  boxShadow="3px 3px 5px 6px var(--amplify-colors-neutral-60)"
+  color="var(--amplify-colors-blue-60)"
+  height="4rem"
+  maxWidth="100%"
+  padding="1rem"
+  width="20rem"
+  fullWidth="true"
+  (click)="clickbutton('span')"
+>
+  'I'm a span! 🤩
+</span>
+<br />
+<br />
+
+<br />
+
+<div
+  amplify-view
+  ariaLabel="View example"
+  backgroundColor="var(--amplify-colors-white)"
+  borderRadius="6px"
+  border="1px solid var(--amplify-colors-black)"
+  boxShadow="3px 3px 5px 6px var(--amplify-colors-neutral-60)"
+  color="var(--amplify-colors-blue-60)"
+  height="4rem"
+  maxWidth="100%"
+  padding="1rem"
+  width="20rem"
+  fullWidth="true"
+  (click)="clickbutton('div')"
+>
+  'I'm a div! 🤩
+</div>
+<br />
+
+<button
+  amplify-view
+  ariaLabel="View example"
+  backgroundColor="var(--amplify-colors-white)"
+  borderRadius="6px"
+  border="1px solid var(--amplify-colors-black)"
+  boxShadow="3px 3px 5px 6px var(--amplify-colors-neutral-60)"
+  color="var(--amplify-colors-blue-60)"
+  height="4rem"
+  maxWidth="100%"
+  padding="1rem"
+  width="20rem"
+  fullWidth="true"
+  (click)="clickbutton('Button')"
+>
```

Comment:
This looks like it's duplicate of Line 1 to 18. Should we remove this?

Reply:
First one is disabled ( 1 to 18 ) and other one is not disabled.

Reply:
Ahh gotcha!